etcdctl/command/mk_command.go

```diff
@@ -51,7 +51,9 @@ func mkCommandFunc(c *cli.Context, ki client.KeysAPI) {
 	ttl := c.Int("ttl")
 
 	ctx, cancel := contextWithTotalTimeout(c)
-	resp, err := ki.Set(ctx, key, value, &client.SetOptions{TTL: time.Duration(ttl) * time.Second, PrevExist: client.PrevIgnore})
+	// PrevNoExist means that the Node must not exist.
+	// Set method returns error if the key already exists.
```

Comment:
`Set method returns error if the key already exists.` This sentence is not true if Set method doesn't use PrevNoExist.

Also the comments miss the part that why you explain how Set returns error here.